unlock-app/src/components/interface/checkout/main/checkoutMachine.ts

```diff
@@ -272,21 +264,16 @@ export const checkoutMachine = createMachine(
               target: 'RETURNING',
               cond: (_, event) => event.existingMember,
             },
-            {
-              actions: ['selectLock'],
-              target: 'RENEW',
-              cond: (_, event) => event.expiredMember,
-            },
             {
               actions: ['selectLock'],
               target: 'QUANTITY',
-              cond: (_, event) => !event.skipQuantity,
+              cond: (_, event) => !event.skipQuantity && !event.expiredMember,
             },
             {
               actions: ['selectLock'],
               target: 'METADATA',
               cond: (_, event) => {
-                return !event.skipRecipient
+                return !event.skipRecipient && !event.expiredMember
```

Comment:
I was wondering if we should let the user update their metadata went they extend, but that's probably for another PR